README.md

```diff
@@ -7,8 +7,8 @@ The Liberty Config Language Server provides language server features for Liberty
 The Liberty Config Language Server is included in the following client IDEs.
 
 * [Liberty Tools for VS Code](https://github.com/OpenLiberty/liberty-tools-vscode)
```

Comment:
I think this one should be `Liberty Tools for Visual Studio Code` also, right? Thanks for the PR. We didn't catch this earlier.

Reply:
I will make this update after merging.